libvast/src/system/importer.cpp

```diff
@@ -179,34 +179,31 @@ behavior importer(importer_actor* self, path dir, archive_type archive,
                      "(currently unsupported!)");
       return self->state.stg->add_outbound_path(index);
     },
-    [=](exporter_atom, const actor& exporter) {
+    [=](exporter_atom, const caf::actor& exporter) {
       VAST_DEBUG(self, "registers exporter", exporter);
       return self->state.stg->add_outbound_path(exporter);
     },
-    [=](stream<importer_state::input_type>& in) {
+    [=](caf::stream<importer_state::input_type>& in) {
       auto& st = self->state;
       VAST_DEBUG(self, "adds a new source:", self->current_sender());
       st.stg->add_inbound_path(in);
     },
-    [=](add_atom, const actor& subscriber) {
+    [=](add_atom, const caf::actor& subscriber) {
       auto& st = self->state;
       VAST_DEBUG(self, "adds a new sink:", self->current_sender());
       st.stg->add_outbound_path(subscriber);
     },
-    [=](subscribe_atom, flush_atom, actor& listener) {
+    [=](subscribe_atom, flush_atom, caf::actor& listener) {
       auto& st = self->state;
       VAST_ASSERT(st.stg != nullptr);
       st.flush_listeners.emplace_back(std::move(listener));
       detail::notify_listeners_if_clean(st, *st.stg);
     },
-    [=](status_atom) {
-      return self->state.status();
-    },
+    [=](status_atom) { return self->state.status(); },
     [=](telemetry_atom) {
       self->state.send_report();
       self->delayed_send(self, defs::telemetry_rate, telemetry_atom::value);
-    }
-  };
+    }};
```

Comment:
```suggestion
    },};
```

Trailing comma looks better after clang-format.